### ISSUE TITLE
training loss nan

### ISSUE THREAD

**Issue by nchafni** [Neerajj9/CSRNet-keras#26]
loss is nan by second step, any pointers?

**Comment by qq1993106** [comment]
my problem is same to you. do you have done it ?
